os-autoinst-distri-opensuse-deps.spec

```
@@ -81,6 +81,15 @@ Recommends:     os-autoinst-devel
 %description
 Metapackage that contains the dependencies of os-autoinst-distri-opensuse.
 
+%package worker
+Summary:        Convenience package pulling in os-autoinst-distri-openSUSE dependencies and the openQA worker
+Group:          Development/Tools/Other
+Requires:       %{name} = %{version}
+Requires:       openQA-worker
+
+%description no-selenium-devel
```

I guess it's leftover from something?

```suggestion
%description
```